core/trino-main/src/main/java/io/trino/server/remotetask/DynamicFiltersFetcher.java

```diff
@@ -52,6 +55,7 @@
     private final Duration refreshMaxWait;
     private final Executor executor;
     private final HttpClient httpClient;
+    private final ScheduledExecutorService scheduledExecutor;
```

Comment:
I assume once the dynamic filter version is updated before transitioning a task to `FINISHED` changes to this class won't be necessary.